hudi-common/src/test/java/org/apache/hudi/common/functional/TestHoodieLogFormat.java

```diff
@@ -124,12 +125,13 @@ public static void tearDownClass() {
 
   @BeforeEach
   public void setUp() throws IOException, InterruptedException {
-    this.fs = MiniClusterUtil.fileSystem;
+    FileSystem fileSystemOriginal = MiniClusterUtil.fileSystem;
 
-    assertTrue(fs.mkdirs(new Path(tempDir.toAbsolutePath().toString())));
+    assertTrue(fileSystemOriginal.mkdirs(new Path(tempDir.toAbsolutePath().toString())));
     this.partitionPath = new Path(tempDir.toAbsolutePath().toString());
     this.basePath = tempDir.getParent().toString();
-    HoodieTestUtils.init(MiniClusterUtil.configuration, basePath, HoodieTableType.MERGE_ON_READ);
+    HoodieTableMetaClient init = HoodieTestUtils.init(MiniClusterUtil.configuration, basePath, HoodieTableType.MERGE_ON_READ);
+    this.fs = init.getFs();
```

Comment:
@loukey-lj Could you clarify how the changes improve the tests for #6602  you put up?

Reply:
In the actual running process fs comes from TableMateClinet.So I initialized a TableMateClinetto get fs instead of fileSystemOriginal.

Reply:
Got it.  Could you point out which set of tests fail without your previous fix (#6602) after this PR?  We should add some comments on these tests mentioning that they test the logic around creating new log files based the size configured.   If there are no such test, we should add one.  Let me know if this makes sense.  We can sync up through Hudi Slack.